src/features/ansibleMetaData.ts

```diff
@@ -80,7 +80,7 @@ export class MetadataManager {
         if (ansibleMetaData.ansiblePresent) {
           console.log("Ansible found in the workspace");
           this.cachedAnsibleVersion =
-            ansibleMetaData.metaData["ansible information"]["version"];
+            ansibleMetaData.metaData["ansible information"]["core version"];
```

Comment:
Please also update at line https://github.com/ansible/vscode-ansible/blob/a9ba416bc0948b925f96b6dbe192c62876f8ef38/src/features/ansibleMetaData.ts#L101

Reply:
Thanks for pointing it out!